styles/go.dev/catppuccin.user.css

```diff
@@ -0,0 +1,930 @@
+/* ==UserStyle==
+@name go.dev Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/go.dev
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/go.dev
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/go.dev/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Ago.dev
+@description Soothing pastel theme for go.dev
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('go.dev') {
+  @media (prefers-color-scheme: light) {
+    :root {
```

Comment:
I think you missed the `[data-theme='auto']` part here?

Reply:
Didn't include it as it didn't really affect anything. Would it better to add for clarity?

Reply:
Yeah I would think so. Just looks odd having it on one and not the other.